src/manager_inst_tests.cc

```diff
@@ -1,10 +1,10 @@
 #include "manager_inst_tests.h"
 
 // - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-TestProducer::TestProducer(cyclus::Context* ctx) : cyclus::Facility(ctx) {}
+//TestProducer::TestProducer(cyclus::Context* ctx) : cyclus::Facility(ctx) {}
```

Comment:
I ran `cycamore_unit_tests` with and without these commented out and both times it ran 195 tests... I didn't do a clean build in between so maybe that's why?

Reply:
When these lines are included (not commented out) I got errors when building cycamore because of duplicate definitions. Removing this and line 7 removes the duplication. You most likely didn't run into this issue because it was happening on the build and not with the unit tests. I will remove the commented out lines.

Reply:
I see they were added in `deploy_inst_tests.cc` (along with the corresponding class definition in `deploy_inst_tests.h`) so that is where the `TestProducer` class duplication is coming from, since it was also defined in `manager_inst_tests.h`. 

I'm not sure what these lines do/don't do. I'm guessing it needs to test `DeployInst` as a commodity producer, which is needed, and further, it is likely needed for both `ManagerInst` and `DeployInst`. 

I tried deleting the duplicated definition from `deploy_inst_tests.h`, and including `manager_inst_tests.h` in `deploy_inst_tests.cc`. Not sure that's good practice, though, but it didn't work either way. I'll play around a little more.

Reply:
All of the tests in `manager_insts_tests.cc` are being run without these lines. My guess is that the initialization of the function in `deploy_insts_tests.cc` is still valid for this one.